packages/react-reconciler/src/ReactFiberTransitionPool.old.js

```diff
@@ -0,0 +1,154 @@
+/**
```

Comment:
Is this file intentional? I think it's unused since the contents have been moved to the ReactFiberTransition file.